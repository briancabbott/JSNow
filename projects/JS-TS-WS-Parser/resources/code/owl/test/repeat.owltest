. x .
. x x .
. x x x .
. x x x x .
. x x x x x .
. x x x x x x .
. .
. y, y, y .
. z z z z z .
. z z z z .
. w z w z w .
---
lines = ('.' line '.')*
line =
 'x'{1} : one
 'x'{2, 4} : two-to-four
 'x'{5+} : five-plus
 'y'{','} : list
 'z'{'z', 1+} : odd
 'z' 'z'{'z', 1+} : even
 'w'{'z', 3} : exactly-three
