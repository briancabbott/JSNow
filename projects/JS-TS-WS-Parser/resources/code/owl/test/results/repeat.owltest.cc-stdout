lines (0 - 121)
  line : ONE (2 - 3)
  line : TWO_TO_FOUR (8 - 11)
  line : TWO_TO_FOUR (16 - 21)
  line : TWO_TO_FOUR (26 - 33)
  line : FIVE_PLUS (38 - 47)
  line : FIVE_PLUS (52 - 63)
  line : LIST (67 - 67)
  line : LIST (72 - 79)
  line : ODD (84 - 93)
  line : EVEN (98 - 105)
  line : EXACTLY_THREE (110 - 119)
