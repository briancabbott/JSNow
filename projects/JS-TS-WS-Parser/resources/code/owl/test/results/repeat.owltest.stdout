. . x        .
    line:one  

. . x x              .
    line:two-to-four  

. . x x x            .
    line:two-to-four  

. . x x x x          .
    line:two-to-four  

. . x x x x x      .
    line:five-plus  

. . x x x x x x    .
    line:five-plus  

. .          .
   line:list  

. . y, y, y   .
    line:list  

. . z z z z z .
    line:odd-  

. . z z z z   .
    line:even  

. . w z w z w          .
    line:exactly-three  
